modules/beamdyn/src/Registry_BeamDyn.txt

```diff
@@ -44,7 +44,6 @@ typedef   ^        InitOutputType CHARACTER(ChanLen)  WriteOutputUnt {:}  -   -
 typedef   ^        InitOutputType ProgDesc            Ver             -   -   -   "This module's name, version, and date"  -
 typedef   ^        InitOutputType R8Ki                kp_coordinate  {:}{:} - -   "Key point coordinates array"            -
```

Comment:
can `kp_coordinate` and `kp_total` be removed here, too?

Reply:
Done!